### PR TITLE
feat: new audit styles

### PR DESCRIPTION
**Pull request by patrickhulce**
since the devtools folks will be picking up next week figured I'd throw up what I had locally before they start. it wasn't a ton of work lost if we just close this

## Mock
image

## Impl so far

image

### CODE REVIEW

**Comment by ebidel** [comment]
I chatted with @paulirish about this offline, but we both agreed that some of the new DevTools UI mocks are kinda lack luster. The list in particular is really plain. Not very inspiring to motivate a developer.

If devtools does end up going with a more generic UI, I wonder if we can diverge a bit with a different stylesheet that DT loads? Small things like this should be pretty easy to tweak.

**Comment by patrickhulce** [comment]
Yeah fair enough, it's easy for devtools to drop in styles like these anyway :)

**Comment by paulirish** [comment]
wait what happened here. this PR was pretty awesome. 

let's bring it back?

**Comment by paulirish** [comment]
@ebidel re your comment. I agreed with you that X's on the left are superior. :)
That was mostly it. This PR didnt change that and generally improved things. I think we should apply this, even if we'll do another pass on visuals.

File: lighthouse-core/report/v2/renderer/category-renderer.js [hunk 1]
```diff
@@ -56,7 +56,6 @@ class CategoryRenderer {
 
     // Append audit details to header section so the entire audit is within a <details>.
     const header = /** @type {!HTMLDetailsElement} */ (this._dom.find('.lh-score__header', tmpl));
-    header.open = audit.score < 100; // expand failed audits
```
Comment (paulirish) [inline lighthouse-core/report/v2/renderer/category-renderer.js]: looks like debugText is printed even if the header isn't `open`?

this header only reveals helpText and any `details` stuff?

File: lighthouse-core/report/v2/report-styles.css [hunk 2]
```diff
@@ -182,53 +181,54 @@ summary {
...
   content: none;
 }
 
 .lh-score__value--pass {
-  background: var(--pass-color);
+  color: var(--pass-color);
 }
 
 .lh-score__value--pass::after {
-  background: url('data:image/svg+xml;utf8,<svg width="12" height="12" viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg"><title>pass</title><path d="M9.17 2.33L4.5 7 2.83 5.33 1.5 6.66l3 3 6-6z" fill="white" fill-rule="evenodd"/></svg>') no-repeat 50% 50%;
```
Comment (ebidel) [inline lighthouse-core/report/v2/report-styles.css]: Can we keep the svg icons? They look cleaner than the ascii. You can change the size and `fill` color as needed.

**Comment by ebidel** [comment]
@paulirish can we at least remove some of the lines? There are so. many. lines. and a lot of white. Not much to break things up anymore.

What about using a smaller guage for the sore?

File: lighthouse-core/report/v2/renderer/category-renderer.js (hunk 1 above)
Comment (patrickhulce) [inline lighthouse-core/report/v2/renderer/category-renderer.js]: yup, that was unchanged do we want to stick debugText in details?

File: lighthouse-core/report/v2/report-styles.css (hunk 2 above)
Comment (patrickhulce) [inline lighthouse-core/report/v2/report-styles.css]: done, looks much sharper 👌

**Comment by patrickhulce** [comment]
mini-gauges lookin' ✈️ 
image

**Comment by ebidel** [comment]
It's the little things. This looks muuuuuch better!

**Comment by ebidel** [comment]
@patrickhulce looks like tests need adjusting.

**Comment by patrickhulce** [comment]
adjusted

**Pull request merged**